sdk/communication/azure-communication-phonenumbers/test/test_sip_routing_client_e2e_async.py

```diff
@@ -4,197 +4,192 @@
 # license information.
 # --------------------------------------------------------------------------
 
-import asyncio
-
 from _shared.asynctestcase import AsyncCommunicationTestCase
 from _shared.helper import URIReplacerProcessor
 from _shared.utils import async_create_token_credential, get_http_logging_policy
+from sip_routing_helper import DomainReplacerProcessor, get_user_domain, trunks_are_equal, routes_are_equal
 
 from azure.communication.phonenumbers.siprouting.aio import SipRoutingClient
 from azure.communication.phonenumbers.siprouting._generated.models import SipTrunkRoute
 from azure.communication.phonenumbers.siprouting._models import SipTrunk
 from azure.communication.phonenumbers._shared.utils import parse_connection_str
 
 class TestSipRoutingClientE2EAsync(AsyncCommunicationTestCase):
-    TRUNKS = [SipTrunk(fqdn="sbs1.sipconfigtest.com", sip_signaling_port=1122), SipTrunk(fqdn="sbs2.sipconfigtest.com", sip_signaling_port=1123)]
-    ROUTES = [SipTrunkRoute(name="First rule", description="Handle numbers starting with '+123'", number_pattern="\\+123[0-9]+", trunks=["sbs1.sipconfigtest.com"])]
+    user_domain = get_user_domain()
```

Comment:
The same comments left in the previous file also apply to this one.

Reply:
Fixed.